<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TemperatureCalculatorClass</class>
 <widget class="QMainWindow" name="TemperatureCalculatorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>609</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TemperatureCalculator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>176</y>
      <width>401</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="TopHalfLayout">
       <item>
        <widget class="QTextEdit" name="InputBox"/>
       </item>
       <item>
        <widget class="QPushButton" name="StartingFButton">
         <property name="text">
          <string>°F</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="StartingCButton">
         <property name="text">
          <string>°C</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="StartingKButton">
         <property name="text">
          <string>°K</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="BottomHalfLayout">
       <item>
        <widget class="QLabel" name="ConvertedValueLabel">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>Converted value</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="FButton">
         <property name="text">
          <string>°F</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="CButton">
         <property name="text">
          <string>°C</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="KButton">
         <property name="text">
          <string>°K</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>609</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="TemperatureCalculator.qrc"/>
 </resources>
 <connections/>
</ui>
